### PR TITLE
E9996 documentation

### PR DESCRIPTION
**Pull request by Jasminewwwu**
The documentation for E9996

### CODE REVIEW

**Comment by Jasminewwwu** [comment]
There is something wrong with the file changed...
I will correct that later

File: index.md [hunk 1]
```diff
@@ -353,7 +353,7 @@ print("\\\\\d{3}")  # Error on this line
 
 ### R0204: Redefined variable type {#R0204}
 
-This error occurs when the type of a variable changes inside a method or a 
+This error occurs when the type of a variable changes inside a method or a
 function. See the examples below.
 
 ```python
```
Comment (nigef) [inline index.md:359]: @Jasminewwwu can we replace these codeblocks with our newer `include` syntax?

Comment (Jasminewwwu) [inline index.md]: sure!

Comment (Jasminewwwu) [inline index.md:359]: what should I change to?

Comment (nigef) [inline index.md:359]: you can see some examples of the includes in this PR:

https://github.com/pyta-uoft/website/pull/16/files

Comment (Jasminewwwu) [inline index.md:359]: I just changed it. Is it correct now?

**Pull request closed without merging**